src/jets.cxx

```diff
@@ -765,6 +765,106 @@ Btagging(ROOT::RDF::RNode df,
     return df2;
 }
 } // end namespace scalefactor
+
+namespace quantity {
+
+/**
+ * @brief Applies jet identification criteria based on JSON-defined jet ID corrections.
+ *
+ * This function loads jet ID definitions from correctionlib JSON files for the specified
+ * jet collection and evaluates two sets of criteria:
+ *  - Tight ID
+ *  - Tight Lepton Veto ID
+ *
```

Comment:
Can you move the quantity namespace before the scalefactor namespace, so that it matches the order in the .hxx file.